client/src/Helper.ts

```diff
@@ -111,11 +111,13 @@ export class Helper {
     return Helper.isWin ? "\\Gobra\\GobraTools" : "/Gobra/GobraTools"
   }
 
-  private static getJavaHome(): Promise<string> {
+  private static getJavaHome(): Promise<IJavaHomeInfo> {
     return new Promise((resolve, reject) => {
       try {
         const options = {
-          version: ">=1.8"
+          version: ">=1.8",
+          mustBe64Bit: true,
+          mustBeJDK: true // we currently disallow JREs
```

Comment:
Why do we disallow JREs?

Reply:
This is a recommendation by Federico and they do the same for Prusti